app/configurator/interactive-filters/interactive-filters-configurator.tsx

```diff
@@ -22,6 +22,8 @@ import { useLocale } from "@/locales/use-locale";
 
 import { getTimeSliderFilterDimensions } from "./helpers";
 
+const ENABLE_TIME_SLIDER = false;
```

Comment:
You can use `app/configurator/components/flag.tsx` to have flags that can be dynamically set via the URL :)

Reply:
That's a really neat way, thanks 💯